configurations/test/env.yml

```diff
@@ -1,3 +1,9 @@
+BUGSNAG_PROJECT_NOTIFIER_API_KEY: notifier key
```

Comment:
Rename this file to `env.yml.tmp`

Reply:
Could you order the params alphabetically please? Also insert blank lines between the Auth0 params and the Mongo params.